docs/usage/setup-azure-devops.md

```diff
@@ -114,7 +114,7 @@ always-auth=true
 
 ### Add renovate.json file
 
-Additionally, you can create a `renovate.json` file which holds the Renovate configuration, in the root of the repo.
+Additionally, you can create a `renovate.json` file which holds the Renovate configuration, in the root of the repo to update.
```

Comment:
```suggestion
Additionally, you can create a `renovate.json` file (which holds the Renovate configuration) in the root of the repository you want to update.
```

I think this might be even better. 😄 What do you think?

Reply:
💯